apps/src/templates/sectionProgressV2/progress-table-v2.module.scss

```diff
@@ -5,6 +5,21 @@
   background-color: white;
 }
 
+@mixin remove-default-button-styles {
```

Comment:
Ahhh... this is the stuff I have NOT seen in our codebase much.  I would love to have some more conversations around standardizing use of it - perhaps accessibility is a great place to start???  It seems like an obvious use for @mixins :)